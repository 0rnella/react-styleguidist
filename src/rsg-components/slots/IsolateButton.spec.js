import React from 'react';
import IsolateButton from './IsolateButton';

it('should renderer a link to isolated mode', () => {
	const actual = shallow(<IsolateButton name="Pizza" />);

	expect(actual).toMatchSnapshot();
});

it('should renderer a link home in isolated mode', () => {
	const actual = shallow(<IsolateButton name="Pizza" isolated />);

	expect(actual).toMatchSnapshot();
});
