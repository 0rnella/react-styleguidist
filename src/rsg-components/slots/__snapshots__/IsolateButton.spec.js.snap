// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should renderer a link home in isolated mode 1`] = `
<Styled(ToolbarButton)
  href="blank"
  title="Show all components"
>
  <MdFullscreenExit />
</Styled(ToolbarButton)>
`;

exports[`should renderer a link to isolated mode 1`] = `
<Styled(ToolbarButton)
  href="blank#!/Pizza"
  title="Open isolated"
>
  <MdFullscreen />
</Styled(ToolbarButton)>
`;
